show_client/show_paths.go

```diff
@@ -82,4 +82,14 @@ func init() {
 		nil,
 		showCmdOptionInterface,
 	)
+	sdc.RegisterCliPath(
+		[]string{"SHOW", "buffer_pool", "watermark"},
+		getBufferPoolWatermark,
+		nil,
+	)
+	sdc.RegisterCliPath(
+		[]string{"SHOW", "buffer_pool", "persistent-watermark"},
+		getBufferPoolPersistentWatermark,
+		nil,
```

Comment:
non blocking comment: Please add namespace option (unimplemented) to these paths in your next PR